activity/actreturn/README.md

```diff
@@ -0,0 +1,50 @@
+# tibco-reply
+This activity provides your flogo application the ability to reply to a trigger invocation.
+
+## Installation
+
+```bash
+flogo add activity github.com/TIBCOSoftware/flogo-contrib/activity/reply
```

Comment:
This should be flogo install github.com/TIBCOSoftware/flogo-contrib/activity/actreturn?